### ISSUE TITLE
Clan Names

### ISSUE THREAD

**Issue by MR-P4NDA** [phen0menon/poe-trade-telegram-notifier#4]
for a brief moment the reply started to work for me. However, when testing it couldn't reply to users that had a clan name. 

e.g. 

From <MoH©> 相も変わらずお人好しで格好つけてばかりだね:

def extract_name_from_whisper(whisper: str) -> str:
    parts = whisper.split(":")
    if len(parts) > 1:
        name = parts[0].split()[1].strip()
        return name
    return whisper

Looks like it can extract either the username or the clan name, but can't differentiate between the two. This causes whispers to people with clan names to fail (because the whisper is trying to @<Clan Name>, rather than @username).

in the above example, the username extracted was <MoH©> and when attempting to reply, it would try to whisper @<MoH©> rather than 相も変わらずお人好しで格好つけてばかりだね

**Comment by phen0menon** [comment]
Wow! That's really a good catch. This may be a cause to the previous issue.

I will dig into this at the end of the day.

**Comment by MR-P4NDA** [comment]
Think I fixed the issue with clan names. Haven't pushed since I don't think I have permissions. I tested with clan names and without.. Again, suddenly the app started to allow replies randomly today, once I realized, I got on and started doing some testing. It seems like the ability to reply is intermittent (more not working than does). After a VERY long period of being on (since 8AM CST), the observer starts to work. 

Here's the code:

`def extract_name_from_whisper(whisper: str) -> str:
    parts = whisper.split(":")
    if len(parts) > 1:
        name = parts[-2].split()[-1].strip()
        return name
    return whisper`
